tracer/build/_build/Build.GitHub.cs

```diff
@@ -710,7 +713,6 @@ LabbelerConfiguration GetLabellerConfiguration()
        .DependsOn(CreateRequiredDirectories)
        .Requires(() => AzureDevopsToken)
        .Requires(() => Version)
-       .Requires(() => TargetBranch)
```

Comment:
I believe it would be better to separate concerns here and not force 2 use cases into one.
I'd rather:
- Rename the `DownloadAzurePipelineAndGitlabArtifacts` into `DownloadReleaseArtifacts` and keep the requirement of `TargetBranch`
- Create a new target `DownloadAzurePipelineFromBuild` that would have a BuildId requirement.

And obviously, this second target would only call a function that could be reused in `DownloadReleaseArtifacts` as part of the code is common indeed.

Reply:
This is good feedback, will do!

Reply:
I separated the two concerns in 2857d4e53f9a64291c46b04b5b667e39232d6ccd and renamed `DownloadAzurePipelineAndGitlabArtifacts` to `DownloadReleaseArtifacts` in a63d04ad7178107a2db635ee286b0b893ab7187a